test/tutorial.py

```diff
@@ -6,9 +6,25 @@
 dealer = Dealer(deck)
 
 # Create players for your game
-matthew = Player('Matthew')
-mark = Player('Mark')
-players = [matthew, mark]
+
+name = input("Please enter your name: ")
+#runs untill else#
+while True:
+        #asks to confirm must use Y or N
+    confirm = input("CONFIRM "+ name + "? Please enter 'Y/N':")
+        #IF Y Print and end the loop
+    if confirm =="Y":
+        print("Awesome your username is " + name)            
+        break
+        #if N then ask them to re enter their name
+    if confirm =="N":
+        print("You have selected 'NO', please select another name....")
+        name = input("Please re-enter your desired name: ")
+        #if its anything else besides a Y or N make them re type their responce
+    else:
+        print("Looks like there was a typo in your responce, please try again and remember to use Capital letters.")
+
+players= [Player(name)]
```

Comment:
This test seems to operate well! It is simple and does what it is intended to do. If you want to improve it further, you can add more error handling to handle unexpected inputs from the user.

Reply:
Thanks! It takes all kinds of responses and only looks for 2 responses, and if it isn't those responses, then it won't crash but prompt them to try again 👍